src/sources/host_metrics.rs

```diff
@@ -692,6 +694,23 @@ fn add_collector(collector: &str, mut metrics: Vec<Metric>) -> Vec<Metric> {
     metrics
 }
 
+pub async fn init_roots() {
+    #[cfg(target_os = "linux")]
+    {
+        let virt = heim::virt::detect().await;
+        if let Some(virtualized) = virt {
+            if virtualized.is_container() {
+                if let Some(procfs_root) = std::env::var_os("PROCFS_ROOT") {
+                    heim_runtime::linux::set_procfs_root(PathBuf::from(&procfs_root));
```

Comment:
Let's add a debug level print here (and below) to notify users that the non-standard path is picked up for use.

Reply:
In my local testing, I figured this branch is likely not executing at all, presumably because the `heim::virt::detect()` is broken.